orga/tests/acceptance/student-list-test.js

```diff
@@ -104,10 +104,33 @@ module('Acceptance | Student List', function(hooks) {
 
           // when
           await visit('/eleves');
-          
+
           // then
           assert.dom('.button').hasText('Importer');
         });
+
+        test('it should display success message and reload students', async function(assert) {
+          // given
+          user = createUserManagingStudents('OWNER');
+          await authenticateSession({
+            user_id: user.id,
+          });
+          await visit('/eleves');
+
+          // when
+          await triggerEvent(
+            'input[type=file]',
+            'change',
+            { files: [new Blob(['validFile'])] }
+          );
+
+          // then
+          assert.dom('.alert-zone--success').exists();
+          assert.dom('.alert-zone--success').hasText('Les élèves ont été importés avec succès.');
+          assert.dom('.table tbody tr').exists({ count: 1 });
+          assert.dom('.table tbody tr td:first-child').hasText('Cover');
+          assert.dom('.table tbody tr td:nth-child(2)').hasText('Harry');
+        });
```

Comment:
On teste l'affichage en cas de succès mais pas en cas d'erreur

Reply:
Impossible de tester en cas d'erreur, puisqu'ici on "triche" par un blob.

Reply:
Ou du moins, j'y ai passé 1h, pas réussi. J'ai estimé que c'était pas worse it.

Reply:
English nazi -> worth it